testing/kuttl/e2e/delete/README.md

```diff
@@ -1,18 +1,18 @@
 ### Delete test
 
-#### Regular cluster delete
+#### Regular cluster delete (00-01)
 
 * Start a regular cluster
 * Delete it
 * Check that nothing remains.
 
-#### Delete cluster with replica
+#### Delete cluster with replica (10-11)
 
 * Start a regular cluster with 2 replicas
 * Delete it
 * Check that nothing remains
 
-#### Delete a cluster that never started
+#### Delete a cluster that never started (20-21)
```

Comment:
Do we need to check this? Kube should cleanup the resources regardless of the cluster status

Reply:
Hmmm, I think this test was to check that we didn't have a finalizer on the cluster that made it impossible to delete while not running.